### ISSUE TITLE
Aero-Ion theme crashes on Windows

### ISSUE THREAD

**Issue by bb30994** [FoldingAtHome/fah-control#34]
(same as issue #16?)

On Win7 64-bit OS, in the Advanced Control, if I go to Preferences and choose the "Aero-Ion" theme, the Advanced Control will crash immediately.

**Comment by bb30994** [comment]
(quot: toTOW) Same issue on Windows 10 64 bits.

**Comment by Djfe** [comment]
I can confirm this issue

**Comment by docquantum** [comment]
I can also confirm

**Comment by sombraguerrero** [comment]
There's a crash log in French attached to the forum post about this, but I'll attach mine from Windows 10 64-bit, in English.
`Faulting application name: FAHControl.exe, version: 0.0.0.0, time stamp: 0x00000000
Faulting module name: KERNELBASE.dll, version: 10.0.17134.441, time stamp: 0x3da51fd0
Exception code: 0x80000003
Fault offset: 0x0018b1b2
Faulting process id: 0x105c
Faulting application start time: 0x01d4a3be6c5b84c2
Faulting application path: D:\Program Files (x86)\FAHClient\FAHControl.exe
Faulting module path: C:\Windows\System32\KERNELBASE.dll
Report Id: 8dca5199-24ba-4de2-bd58-91d8bee3a334
Faulting package full name: 
Faulting package-relative application ID: `

**Comment by clockworkmcd** [comment]
getting the same here. any idea how or where to go into configuration files and change it back so we can run it again?

**Comment by thedarkfalcon** [comment]
I went to make a change, but it looks like this theme was [removed ](https://github.com/FoldingAtHome/fah-control/commit/82c77f027376254e82899b8311e070bace9b9ee9)from Master about a week ago, now we just wait for the next release to come out to actually fix this.

@clockworkmcd I actually just deleted the dodgy theme directory in _%AppData%\Roaming\FAHClient\themes_ and it seemed to revert back to the default one the next time I tried opening the program. I'm not sure where the config file which stores the state for things like themes is stored.
